src/talk/renderer/Settings/DesktopSettingsSection.vue

```diff
@@ -66,6 +86,40 @@ function relaunch() {
 			<NcCheckboxRadioSwitch :checked.sync="systemTitleBar" type="switch">
 				{{ t('talk_desktop', 'Use system title bar') }}
 			</NcCheckboxRadioSwitch>
+
+			<SettingsFormGroup :label="t('talk_desktop', 'Zoom')">
+				<template #icon="{ size }">
+					<IconMagnify :size="size" />
+				</template>
+				<template #description>
+					<!-- eslint-disable-next-line vue/no-v-html -->
+					<span v-html="zoomHint" />
+				</template>
+				<template #default="{ inputId, descriptionId }">
+					<NcButton type="tertiary" @click="zoomFactor /= zoomStep">
+						<template #icon>
+							<IconMinus :size="20" />
+						</template>
+					</NcButton>
+					<NcTextField :id="inputId"
+						class="zoom-input"
+						:aria-describedby="descriptionId"
+						inputmode="number"
+						:value="zoomFactorPercentage"
+						@change="zoomFactorPercentage = $event.target.value" />
```

Comment:
Not a big deal, but should we handle border values here (NcTextField, NcButton)?

Reply:
It is handled by `zoomFactorPercentage`

Reply:
I meant that it's still possible to set out of range values in UI, they just not kept in config and not reflected in the app window

Reply:
Fixed

Reply:
Yup, works better with manual input. Maybe same getter/setter could be done for `zoomFactor`?
![2024-10-29_16h43_11](https://github.com/user-attachments/assets/5dfebd88-504f-4faf-afa6-5fce59d3c907)

Reply:
It has different limits. Will put it to constants